config.py

```
@@ -29,415 +21,270 @@
        if not BITRATE >= 48000:
-          BITRATE=48000
+          BITRATE = 48000
     else:
-       BITRATE=48000
+       BITRATE = 48000
     
     if E_FPS:
        try:
-          FPS=int(E_FPS)
+          FPS = int(E_FPS)
        except:
```

Ruff (0.15.18)

[error] Do not use bare `except`

(E722)